tools/ocean/divandfull.xml

```diff
@@ -0,0 +1,111 @@
+<tool id="divand_full_analysis" name="DIVAnd" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
+    <description>Data-Interpolating Variational Analysis in n dimensions</description>
+    <macros>
+        <token name="@TOOL_VERSION@">0.1.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="1.8.5">julia</requirement>
+        <requirement type="package" version="2.7.9">julia-divand</requirement>
+        <!--<requirement type="package" version="0.8.2">julia-pycall</requirement>
+        <requirement type="package" version="0.4.4">julia-dates</requirement>
+        <requirement type="package" version="@TOOL_VERSION@">ncdatasets</requirement>
+        <requirement type="package" version="@TOOL_VERSION@">physocean</requirement>
+        -->
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    ## The HOME .julia folder is not writable inside the Docker container, so we need to set one that is writable. 
+    export JULIA_DEPOT_PATH="\$PWD:\$JULIA_DEPOT_PATH" &&
+            julia
+            '$__tool_directory__/divandfull.jl'
+            '$input_netcdf_identifier'
+            '$longmin'
+            '$longmax'
+            '$latmin'
+            '$latmax'
+            '$startdate'
+            '$enddate'
+            '$varname'
+            '$selmin'
+            '$selmax'
+    ]]></command>
+    <inputs>
+        <param argument="input_netcdf_identifier" type="data" format="netcdf" label="Input your netcdf data"/>
+        <param argument="longmin" type="float" min="-180" max="180" value="0" label="Longitude minimal"/>
+        <param argument="longmax" type="float" min="-180" max="180" value="0" label="Longitude maximal"/>
+        <param argument="latmin" type="float" min="-180" max="180" value="0" label="Latitude minimal"/>
+        <param argument="latmax" type="float" min="-180" max="180" value="0" label="Latitude maximal"/>
+        <param argument="startdate" type="text" value="yyyy-mm-dd" label="Input the starting date">
+            <sanitizer invalid_char="">
+                <valid initial="string.letters,string.digits">
+                    <add value="_" />
+                </valid>
+            </sanitizer>
+        </param>
+        <param argument="enddate" type="text" value="yyyy-mm-dd" label="Input the ending date">
+            <sanitizer invalid_char="">
+                <valid initial="string.letters,string.digits">
```

Comment:
why letters? Are you sure you need `-` in your example in the comment you left out the `-`